core/src/test/scala/org/apache/spark/deploy/ExternalShuffleServiceSuite.scala

```diff
@@ -30,13 +28,12 @@ import org.scalatest.concurrent.Eventually
 import org.scalatest.matchers.should.Matchers._
 import org.scalatest.time.SpanSugar._
 
+import org.apache.spark.{ShuffleSuite, SparkContext, SparkException, TestUtils}
+import org.apache.spark.deploy.DeployMessages.ApplicationRemoveTest
 import org.apache.spark.internal.config
-import org.apache.spark.network.TransportContext
-import org.apache.spark.network.netty.SparkTransportConf
-import org.apache.spark.network.server.TransportServer
-import org.apache.spark.network.shuffle.{ExecutorDiskUtils, ExternalBlockHandler, ExternalBlockStoreClient}
+import org.apache.spark.network.shuffle.{ExecutorDiskUtils, ExternalBlockStoreClient}
 import org.apache.spark.storage.{RDDBlockId, ShuffleBlockId, ShuffleDataBlockId, ShuffleIndexBlockId, StorageLevel}
-import org.apache.spark.util.{ThreadUtils, Utils}
+import org.apache.spark.util.ThreadUtils
 
 /**
  * This suite creates an external shuffle server and routes all shuffle fetches through it.
```

Comment:
This comment is now slightly outdated if we're now standing up the ESS within the Worker.